auth/utils/helpers.ts

```diff
@@ -91,20 +91,6 @@ export function getCurrentTime(): number {
   return Math.floor(Date.now() / 1000);
 }
 
-/**
- * Checks for only one of two elements being defined.
- * Returns true if a is defined and b is undefined,
- * or vice versa. Returns false if both are defined
- * or both are undefined.
- *
- * @param a - The first object
- * @param b - The second object
- * @returns true if and only if exactly one of a or b is defined
- */
-export function onlyOne(a: any, b: any): boolean {
-  return Boolean((a && !b) || (b && !a));
-}
-
```

Comment:
I just moved this to be further down in the file, next to the other related functions.